_pages/releases.md

```diff
@@ -9,6 +9,6 @@ permalink: /releases
 # Releases
 
 {% for article in site.data.releases %}
-<p>{{ article.date }} <br>
-<em><a href='{{article.link}}'>{{article.codebase}} - {{article.version}}</a></em></p>
+<p><em><b>{{article.codebase}} - {{article.version}}</b> ({{ article.date | date: "%-d %B %Y" }})</em></p>
```

Comment:
the date formatting diverges here. [perhaps a simple include would make this easy to uniform]